tests/Css/StyleTest.php

```diff
@@ -160,4 +160,25 @@ public function testContent(string $value, $expected): void
         $style->set_prop("content", $value);
         $this->assertSame($expected, $style->content);
     }
+
+    public function valueCaseProvider(): array
+    {
+        return [
+            ["width", "Auto",           "width", "auto"],
+            ["list-style-type", "A",    "list_style_type", "A"],
```

Comment:
Adding some tests is nice! The underscore variant works for both getter and `set_prop`, would be just 3 parameters.

Reply:
But `set_prop` is called with dashes, so that's how I'd test it. In case there's an exception for `list-style-type` or `font-family` or something later.

Reply:
You have a point there. It’s fine for me either way.